cli addon status (#767)
oam-dev/kubevela.io: docs/end-user/components/more.md

By default, the place for installation is specified as control plane cluster or managed cluster inside the metadata of addon. 

> * Some addons support setting parts of parameter while enabling. For example `velaux` addon supports change image repository by set `repo` parameter, then you can change the repo address to your own. eg:

Reviewer: ```suggestion
* Some addons support setting parts of parameter while enabling. For example `velaux` addon supports change image repository by setting `repo` parameter, then you can change the repo address to your own. e.g:
```

```shell
vela addon enable velaux repo=<your repo address>
```

### Get addon info

> If you want to check the status of an addon, or get the knowledge of the available parameters and other useful info of an addon, you can use command `addon status`. eg:

Reviewer: ```suggestion
If you want to check the detail status or get more available parameters of an addon, you can use command `addon status`.  For example:
```
